lib/core/test/unit/Cardano/Wallet/DB/SqliteSpec.hs

```diff
@@ -60,6 +64,8 @@ import Cardano.Wallet.Primitive.Types
     )
 import Cardano.Wallet.Unsafe
     ( unsafeRunExceptT )
+import Control.Monad
```

Comment:
you should put `replicateM_` with `unless` under one hood here

Reply:
OK fixed.